### ISSUE TITLE
Experties value is null

### ISSUE THREAD

**Issue by BirenRathod** [BiologicalRecordsCentre/MammalNet-Europe#99]
@kazlauskis Please check and resolve this issue.

It seems like, the Value for "expertise" doesn't pass through when requested by this -> https://www.mammalweb.org/api/biodiv/user/12774

See below logs on the website for more details.
https://european-mammals.brc.ac.uk/en/admin/reports/dblog/event/963574
https://european-mammals.brc.ac.uk/en/admin/reports/dblog/event/963575
https://european-mammals.brc.ac.uk/en/admin/reports/dblog/event/963576

Possible culprit is this -> Warning: Undefined property: stdClass::$expertise in Drupal\imammalia_extras\Plugin\QueueWorker\GetMwExperience->getExpertise() (line 47 of /code/web/modules/custom/imammalia_extras/src/Plugin/QueueWorker/GetMwExperience.php)
This is the same on the Drupal 9 website too.

**Comment by BirenRathod** [comment]
I'm adding @johnvanbreda, so he can draw some lights here.

**Comment by johnvanbreda** [comment]
@DavidRoy do you know who I should contact about the API to mammalweb.org? Is it still Helen Chappell? The API is returning record not found when we request the details of a user for the ID that mammalweb.org previously gave us.

**Comment by DavidRoy** [comment]
@GrahamCSmith can you advise?

**Comment by GrahamCSmith** [comment]
@johnvanbreda @DavidRoy 
Yes Helen is still the MammalWeb tech person. When I log into iMammalia verification, I can see an up-to-date listing of my MammalWeb testing scores ( I did the Emirates test recently which appears on iMammalia). But I think I'm one of very few people who have scores.

**Comment by kazlauskis** [comment]
@BirenRathod I don't think this "expertise" issue relates to the app.

**Comment by BirenRathod** [comment]
@kazlauskis yes, that is right. I think @johnvanbreda got the root of the problem. Thanks for checking this.
